pandas/io/parsers.py

```diff
@@ -119,9 +119,9 @@
     ``skip_blank_lines=True``, so ``header=0`` denotes the first line of
     data rather than the first line of the file.
 names : array-like, optional
-    List of column names to use. If file contains no header row, then you
-    should explicitly pass ``header=None``. Duplicates in this list are not
-    allowed.
+    List of column names to use. If the file contain a header row,
```

Comment:
Sorry minor typo from me should be ‘file contains’

Reply:
Oops. Fixed, thank you.